pilot/cmd/pilot-discovery/main.go

```diff
@@ -77,6 +77,8 @@ func init() {
 			bootstrap.KubernetesRegistry, bootstrap.ConsulRegistry, bootstrap.EurekaRegistry, bootstrap.CloudFoundryRegistry, bootstrap.MockRegistry))
 	discoveryCmd.PersistentFlags().StringVar(&serverArgs.Config.CFConfig, "cfConfig", "",
 		"Cloud Foundry config file")
+	discoveryCmd.PersistentFlags().StringVar(&serverArgs.Config.ClusterStoreDir, "clusterStoreDir", "",
```

Comment:
registriesConfigDir ? Or something else using registries instead of cluster.

Reply:
(I'm expecting we'll support multiple cloud foundry and all other types of registries)

Reply:
Per discussions with @gnirodi, long term goal is to use cluster-registry https://github.com/kubernetes/cluster-registry. Since this is cluster related, Would ClusterRegistriesDir sound better?